microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -491,22 +492,17 @@ This object represents the original definition of the widget, found in the PWA m
 
 This object is passed as an argument to listeners of service worker widget events of type `widgetinstall`, `widgetuninstall`, and `widgetresume`.
 
-For the `widgetinstall` and `widgetuninstall` event types, the `widgetEvent` object has the following properties:
+For the `widgetinstall`, `widgetuninstall`, and `widgetresume` event types, the `widgetEvent` object has the following properties:
 
 | Property | Description | Type |
 |:--- |:--- |:--- |
-| `widget` | The widget instance that triggered the event. | [widgetInstance](#widgetinstance-object) |
+| `widget` | The widget instance that triggered the event. | [widget](#widget-object) |
 | `instanceId` | The widget instance ID. | `String` |
-
-For the `widgetresume` event type, the `widgetEvent` object has the following property:
-
-| Property | Description | Type |
-|:--- |:--- |:--- |
 | `hostId` | The widget host ID. | `String` |
 
 #### widgetClickEvent object
```

Comment:
[Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2858#issuecomment-1754035166) from Amanda to @captainbrosset in PR main page - 

> Would you PTAL? I'm not sure about the wording of the WidgetClick event bit at the end, so lmk if you have suggestions on how to better word that.